findContact/filterConfig.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const filterConfig = [
+  {
+    label: <FormattedMessage id="ui-plugin-find-contact.contact.status" />,
+    name: 'inactive',
+    cql: 'inactive',
+    values: [
+      {
+        name: 'active',
+        cql: 'false',
+        displayName: <FormattedMessage id="ui-plugin-find-contact.contact.status.active" />,
+      },
+      {
+        name: 'inactive',
+        cql: 'true',
+        displayName: <FormattedMessage id="ui-plugin-find-contact.contact.status.inactive" />,
+      },
+    ],
+  },
+  {
+    label: <FormattedMessage id="ui-plugin-find-contact.contact.categories" />,
+    name: 'categories',
+    cql: 'categories',
+    values: [],
```

Comment:
selection to checkboxes?

Reply:
Basically, I don't expect many categories there. And it's impossible to me for a reasonable time (we don't even have a ticket to fix find contacts plugin) to add support of any filter apart from checkbox - API has been completely changed in SearchAndSortQuery. We have a separate ticket to implement 3-panes-layut based on SandSQ, and I suppose implementing custom filters is included in it (5 SP, https://issues.folio.org/browse/UIF-96)